crates/wash-lib/src/build/component.rs

```diff
@@ -13,8 +13,6 @@ use wasi_preview1_component_adapter_provider::{
     WASI_SNAPSHOT_PREVIEW1_ADAPTER_NAME, WASI_SNAPSHOT_PREVIEW1_REACTOR_ADAPTER,
 };
 use wasm_encoder::{Encode, Section};
-use wit_bindgen_core::Files;
-use wit_bindgen_go::Opts as WitBindgenGoOpts;
```

Comment:
Oh now that these are gone... can we remove the deps?

Do we have to add some sort of warning somewhere to make sure people have the stuff installed/on their `PATH` somewhere? Moving to requiring the extra tool might warrant some extra verbosity around making sure people have it/know where to get it